pkg/gateway/lb/handler/grpc/handler.go

```diff
@@ -960,7 +949,7 @@ func (s *server) MultiInsert(ctx context.Context, reqs *payload.Insert_MultiRequ
 					Owner:        errdetails.ValdResourceOwner,
 					Description:  err.Error(),
 				})
-			log.Debug(err)
+			log.Error(err)
```

Comment:
should we remove this log same as Line#901?
https://github.com/vdaas/vald/pull/1052/files#diff-49d354bb33b23c7ed04a722384dc2e22a9c1e00c2f35537b6ef37b0af0a53f50L901

Reply:
The status code of this error is `INTERNAL`. I think `INTERNAL` errors should be notified to the administrators.